File: U3P2C#/U3P_18.cs
using System;

namespace MyApp // Note: actual namespace depends on the project name.
{
    internal class U3P_18
    {
        static void Main(string[] args)
        {
            double salarioInicial = 1500.0;
            double incrementoAnual = 0.10;
            int numeroAnios = 6;

            Console.WriteLine("Salario inicial: $" + salarioInicial);

            double salarioTotal = salarioInicial;

            for (int anio = 1; anio <= numeroAnios; anio++)
            {
                double salarioAnual = salarioTotal * (1 + incrementoAnual);
                salarioTotal = salarioAnual;
                Console.WriteLine("Salario en el año " + anio + ": $" + salarioAnual);
            }

            Console.WriteLine("Salario al cabo de " + numeroAnios + " años: $" + salarioTotal);
        }
    }
}

File: U3P2J/U3P_18.java
/*
 * Click nbfs://nbhost/SystemFileSystem/Templates/Licenses/license-default.txt to change this license
 */

package com.mycompany.u3p_18;

/**
 *
 * @author alfre
 */
public class U3P_18 {

    public static void main(String[] args) {
        double salarioInicial = 1500.0;
        double incrementoAnual = 0.10;
        int numeroAnios = 6;

        System.out.println("Salario inicial: $" + salarioInicial);

        double salarioTotal = salarioInicial;
        
        for (int anio = 1; anio <= numeroAnios; anio++) {
            double salarioAnual = salarioTotal * (1 + incrementoAnual);
            salarioTotal = salarioAnual;
            System.out.println("Salario en el año " + anio + ": $" + salarioAnual);
        }
        
        System.out.println("Salario al cabo de " + numeroAnios + " años: $" + salarioTotal);
    }
}
